src/libs/geotagging.c

```diff
@@ -97,6 +97,8 @@ typedef struct dt_lib_geotagging_t
   GtkWidget *timezone;
   GList *timezones;
   GtkWidget *gpx_button;
+  GTimeZone *tz_camera;
```

Comment:
Can you comment on this one? tz_camera is setup during module lib init. So it is global to the whole dt session, then I'm wondering how it can be related to any camera?

Reply:
So far there is no relationship between image or camera tz other than manual.
The tz is set manually and has no reason to change during the session if the user doesn't change it, which can happen if he treats images made in different tz.
We could imagine to automate this for images which have actually the tz defined, but that's another story (and if not correctly set in camera could even turns things more complicated).
I hope this clarifies a bit ...

Reply:
Ok, I see. Yes, it clarifies things. Thanks.